addons/fleet/models/fleet_vehicle.py

```diff
@@ -89,7 +89,11 @@ def _compute_manager_id(self):
     net_car_value = fields.Float(string="Purchase Value", help="Purchase value of the vehicle")
     residual_value = fields.Float()
     plan_to_change_car = fields.Boolean(related='driver_id.plan_to_change_car', store=True, readonly=False)
+    plan_to_change_bike = fields.Boolean(related='driver_id.plan_to_change_bike', store=True, readonly=False)
     vehicle_type = fields.Selection(related='model_id.vehicle_type')
+    frame_type = fields.Selection([('diamant', 'Diamant'), ('trapez', 'Trapez'), ('wave', 'Wave')], help="Frame type of the bike")
+    electric_assistance = fields.Boolean()
+    frame_size = fields.Float()
```

Comment:
Those 3 fields should be defined in a migration script (Initialize the columns). 

See: ``create_column`` in https://github.com/odoo/upgrade/blame/master/migrations/util.py#L1818

Thanks.